### ISSUE TITLE
Inconsistent POS and Lemmas in Spanish

### ISSUE THREAD

**Issue by lmorillas** [explosion/spaCy#4921]
## Info about spaCy

* **spaCy version:** 2.2.3
* **Platform:** Linux-4.13.0-46-generic-x86_64-with-glibc2.2.5
* **Python version:** 3.8.0

See this example with single phrases. 

```
import spacy
nlp_es = spacy.load("es_core_news_sm")

def parse(f):
    doc = nlp_es(f)
    return [(x.pos_, x.lemma_) for x in doc]

frases = [
    "Como manzanas todos los días",  # como is a verb: comer
    "Yo comeré manzanas todos los días",  # comeré is a verb: comer
    "A veces como manzanas", # como is a verb: comer
    "Yo como manzanas" # como is a verb: comer
]
for f in frases:
    print(parse(f))
```
And the result:
```
[('SCONJ', 'Como'), ('NOUN', 'manzana'), ('DET', 'todo'), ('DET', 'lo'), ('NOUN', 'día')] 

[('PRON', 'Yo'), ('VERB', 'comer'), ('NOUN', 'manzana'), ('DET', 'todo'), ('DET', 'lo'), ('NOUN', 'día')] 

[('ADP', 'A'), ('NOUN', 'vez'), ('SCONJ', 'comer'), ('NOUN', 'manzana')] 

[('PRON', 'Yo'), ('SCONJ', 'comer'), ('NOUN', 'manzana')]
```
Where `como` is 'SCONJ` and lemmatized as `como` not as `comer`

What is the best way to correct this kind of errors? Should we add exceptions? or is an easy way to train the model with new data? I can't find it in the docs.

**Comment by svlandeg** [comment]
In general, the POS tagger should achieve quite high accuracy (>95%), but ofcourse there will always be **some** mistakes in a Machine Learning model. Looking at the [Spanish models](https://spacy.io/models/es) in particular, you could also experiment whether things improve with `es_core_news_md` instead of `es_core_news_sm`. 

Either way, finetuning on your domain is probably a good idea!
There's a section in the docs specifically on [training the POS tagger](https://spacy.io/usage/training/#example-train-tagger), and I'd recommend you to checkout also the [general training tips](https://spacy.io/usage/training/#tips). When you're retraining/finetuning an existing model, make sure you avoid the probem of [catastrophic forgetting](https://explosion.ai/blog/pseudo-rehearsal-catastrophic-forgetting), which basically means that you have to ensure the model doesn't forget the things it could already do accurately, by mixing in your new training examples with (correct) old predictions or annotations. 

So basically what I would recommend is:
* Spend some time annotating wrong predictions on your specific domain data
* Feed those wrongly predicted instances (with the correct annotation you labeled) back into the model
* Meanwhile, keep measuring the accuracy on a hold-out test set (a part of the dataset you have NOT manually reviewed/corrected) to ensure you're not overfitting on your new examples

Hope that helps :-)
